src/lib/Scenes/Show/Components/Artists/Components/ArtistListItem.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box, Flex, Sans } from "@artsy/palette"
+import InvertedButton from "lib/Components/Buttons/InvertedButton"
+import React from "react"
+
+interface Props {
+  name: string
+  isFollowed: boolean
+  onPress: () => void
+  isFollowedChanging: boolean
+}
+
+export const ArtistListItem: React.SFC<Props> = ({ name, isFollowed, onPress, isFollowedChanging }) => {
+  return (
+    <Flex justifyContent="space-between" alignItems="center" flexDirection="row">
+      <Sans size="3">{name}</Sans>
+      <Box width={112} height={32}>
```

Comment:
We should probably write a todo to remove that hardcoded width

Reply:
good idea, I added a `TODO` here 👍